docs/source/tracking.rst

```diff
@@ -260,6 +260,23 @@ If a run exists when ``autolog()`` captures data, MLflow will log to that run an
   - This feature is experimental - the API and format of the logged data are subject to change.
 
 
+EarlyStopping Integration with Keras Automatic Logging
```

Comment:
Thanks for the detail here, IMO we can put this content into the docstring of `mlflow.keras.autolog()` instead of in this top-level tracking doc as it's pretty specific to Keras autologging & the goal of this doc page is to summarize tracking features at a high-level.